.travis.yml

```diff
@@ -14,6 +14,7 @@ install:
   - conda create -n test-environment python=$TRAVIS_PYTHON_VERSION pandas numpy
   - source activate test-environment
   - conda install --file conda-requirements.txt # xlrd, possibly others
+  - conda install -c ospc taxcalc
```

Comment:
Yes that's good for longer-term.  In the near term we could do `git clone http:/github.com/jdebacker/TaxCalc && cd TaxCalc and git fetch --all && git checkout feature_branch && python setup.py install` , replacing feature_branch with the name of your branch of TaxCalc within your fork.